diff -Naur nginx-1.19.1-origin/src/http/ngx_http_request.c nginx-1.19.1/src/http/ngx_http_request.c
--- nginx-1.19.1-origin/src/http/ngx_http_request.c	2020-07-07 15:56:06.000000000 +0000
+++ nginx-1.19.1/src/http/ngx_http_request.c	2020-07-19 08:38:22.524365796 +0000
@@ -110,9 +110,14 @@
     { ngx_string("Referer"), offsetof(ngx_http_headers_in_t, referer),
                  ngx_http_process_header_line },
 
+	/* 
+		[zairo patch]
+		- Content-Length가 두 개 이상 존재할 경우 Bad Reqeust 처리함
+		- ngx_http_process_unique_header_line을 ngx_http_process_header_line로 치환
+	*/
     { ngx_string("Content-Length"),
                  offsetof(ngx_http_headers_in_t, content_length),
-                 ngx_http_process_unique_header_line },
+                 ngx_http_process_header_line },
 
     { ngx_string("Content-Range"),
                  offsetof(ngx_http_headers_in_t, content_range),
